src/main/java/com/mineboundteam/minebound/magic/UtilitySpells/ShieldUtilitySpell.java

```diff
@@ -41,24 +42,25 @@ public ShieldUtilitySpell(Properties properties, ShieldUtilitySpellConfig config
         this.recovCooldown = config.RECOV_TICKS.get();
     }
 
-    @SubscribeEvent
-    public static void triggerSpell(LivingAttackEvent event) {
-        if (event.getEntityLiving() instanceof Player player && !player.getLevel().isClientSide() && event.getSource() instanceof EntityDamageSource) {
+    @SubscribeEvent(priority = EventPriority.LOW)
+    public static void triggerSpell(LivingHurtEvent event) {
```

Comment:
What is the purpose of changing the priority?

Also, why change this to `LivingHurtEvent`? For the knockback?

Reply:
> What is the purpose of changing the priority.  
This way, if any other mod or event wants to do something with damage before the event is cancelled, those will get to process the method first. For our sake, since we want the telekinetic to capture elytra damage instead of the shield, that one has a higher priority.

I believe LivingHurtEvent is the first in the damage resolution chain, but I could change it to another further down the chain  (I'll just have to do some testing to make sure it works). As for why it's changed from the attack event, the attack event only triggers when an entity damages another

Reply:
Changed to LivingDamagedEvent for this one so that armor and buffs can reduce the amount of damage taken